vmware_rest_code_generator/cmd/refresh_modules.py

```diff
@@ -238,6 +238,14 @@ def gen_documentation(name, description, parameters, added_ins, next_version):
                 ],
                 "type": "str",
             },
+            "vcenter_rest_session_timeout": {
+                "description": [
+                    "Timeout settings for client session. ",
+                    "The maximal number of seconds for the whole operation including connection establishment, request sending and response. ",
+                ],
+                "type": "float",
+                "default": 600,
```

Comment:
I would stay with 300, it's the default and it's enough for all the cases but the OVA export.